requirements.txt

```diff
@@ -1,3 +1,3 @@
-pycrypto == 2.6
+cryptography >= 2.3.1
```

Comment:
Should the SmartOS installation notes about installing the "py27-crypto" pkgsrc package be updated?

Reply:
I replaced that note with a note about installing libsodium and pynacl.